scripts/install.sh

```diff
@@ -185,7 +185,7 @@ install_or_update_system_dependencies()
         # the user might install opkg via the 3rd party moonraker installer script.
         # But in general, PY will already be installed, so there's no need to try.
         # On the K1, the only we thing we ensure is that virtualenv is installed via pip.
-        pip3 install virtualenv
+        pip3 install --trusted-host pypi.python.org --trusted-host pypi.org --trusted-host=files.pythonhosted.org --no-cache-dir virtualenv
```

Comment:
I'll have another look tomorrow.
I dislike the idea of bypassing/disabling the SSL checks for the pypi subdomain.
Still, this can work as a temp fix but should not be added as a permanent solution.

Reply:
> I'll have another look tomorrow. I dislike the idea of bypassing/disabling the SSL checks for the pypi subdomain. Still, this can work as a temp fix but should not be added as a permanent solution.

This is a known issue with K1, it was already the case with wget, no SSL verification works on this OS.

Reply:
Well in that case...